sim/visuals/microbit.ts

```diff
@@ -412,7 +413,7 @@ path.sim-board {
             } else {
                 svg.fills(this.heads.slice(1), theme.accent);
             }
-            if (this.shakeButton) svg.fill(this.shakeButton, theme.virtualButtonUp);
+            if (this.shakeInitialized) svg.fill(this.shakeButton, theme.virtualButtonUp);
```

Comment:
Now shakeButton is assigned in buildDom perhaps no guard is needed here at all. Please see what you think.